[15.0][ADD] project_kanban_form_direct_access (#22)
solvosci/slv-project: project_buttom_kanban_open_form_view/README.rst

====================================

> Proyect buttom kanban open form view

Reviewer: I dont like this name.. BUT it can be... project_direct_access_form_in_kanban_view but dont like too.. @dalonsod what do you propose?

Reviewer 2: Well, try this: `project_kanban_form_direct_access`.

====================================

.. 
   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
   !! This file is generated by oca-gen-addon-readme !!
   !! changes will be overwritten.                   !!
   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
   !! source digest: sha256:fcf7b2d076b18bbde5c5fe1d5a52f68bc39fe98c27ae96d0b526d34fd4801f09
   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!

.. |badge1| image:: https://img.shields.io/badge/maturity-Beta-yellow.png
    :target: https://odoo-community.org/page/development-status
    :alt: Beta
.. |badge2| image:: https://img.shields.io/badge/licence-LGPL--3-blue.png
    :target: http://www.gnu.org/licenses/lgpl-3.0-standalone.html
    :alt: License: LGPL-3
.. |badge3| image:: https://img.shields.io/badge/github-solvosci%2Fslv--project-lightgray.png?logo=github
    :target: https://github.com/solvosci/slv-project/tree/15.0/project_buttom_kanban_open_form_view
    :alt: solvosci/slv-project

|badge1| |badge2| |badge3|

> Add a buttom in kanban view to open the form view

Reviewer: In DESCRIPTION.rst and manifest
```suggestion
Adds direct access to form in kanban view
```
